### PR TITLE
Add mappings for some parameters

### PR DESCRIPTION
**Pull request by elupus**
This adds a few of the mappings from M12676EN document from Nibe. I did not add the mappings from Accessories. I'm pondering if we should grab all fields from that document and add to extensions. But it's unclear which will end up the best in the long run.

Also S320 and S325 seem to be same pump, so adjusted the datafile.

### CODE REVIEW

File: nibe/data/extensions.json [hunk 1]
```diff
@@ -105,5 +105,124 @@
...
+          "0": "Off",
+          "1": "Active",
+          "2": "Passive"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - F2120 slave 4",
+    "files": [],
```
Comment (yozik04) [inline nibe/data/extensions.json:174]: empty array means do not add anywhere?

File: nibe/heatpump.py [hunk 2]
```diff
@@ -33,7 +33,8 @@ class Model(Enum):
     F370 = "f370_f470"
     F470 = "f370_f470"
 
-    S320 = "s350"
```
[the remark this answers is not in the document]
Comment (elupus) [reply nibe/heatpump.py:36]: 320 and 325

File: nibe/data/extensions.json (hunk 1 above)
Comment (elupus) [reply nibe/data/extensions.json:174]: Yes. Seems we have no export with multiple slaves yet

**Comment by yozik04** [comment]
I'd better get at least one S series heatpump working. Then we can polish it all out. If you want to grab something from a document I'd better store it in a separate file then merge all files during convert.

**Pull request merged**